src/templates/releaseNoteLandingPage.js

```diff
@@ -81,43 +81,50 @@ const ReleaseNoteLandingPage = ({ data, pageContext, location }) => {
 
             return (
               <Timeline.Item label={date} key={date}>
-                {posts.map((post) => {
-                  const excerpt = getBestGuessExcerpt(post.mdxAST);
-
-                  return (
-                    <div
-                      key={post.version}
-                      css={css`
-                        margin-bottom: 2rem;
-
-                        &:last-child {
-                          margin-bottom: ${isLast ? 0 : '4rem'};
-                        }
-                      `}
-                    >
-                      <Link
-                        to={post.fields.slug}
+                {posts
+                  .sort((postA, postB) => {
+                    return (
+                      Number(postB.frontmatter.version.replaceAll('.', '')) -
```

Comment:
There are a few release notes that don't have versions attached to them (see [this doc](https://docs.newrelic.com/docs/release-notes/infrastructure-release-notes/kubernetes-integration-release-notes/deprecation-notice-kubernetes-v17-or-lower) or any of the "Cloud integrations" release notes). Does this blow up for those pages?

Reply:
Indeed the build fails for this reason 😄

Reply:
ah I did not realize that, let me add a conditional to this